Week1/github.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+{
+    const URL_Rep = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    function renderRepository(container, repository) {
+        let someDate = new Date(repository.updated_at);
```

Comment:
`someDate` is a meaningful name, for you :smile_cat:, I found this thing actually that we are give a very meaningful names for us but when somebody else reads our code will face an issue understanding what is referring to.

So `someDate` of course it's some date but it would be more clear to call it `lastUpdate` or `latestCommit` or you may have a better name for it.

so the point is this variable handling a date for an event accord in the past, you can call it as the event called simply :slightly_smiling_face: .